mavros_msgs/srv/CommandTOLLocal.srv

```diff
@@ -0,0 +1,10 @@
+# Common type for Take Off and Landing for local (NED) navigation
+
+float32 min_pitch	# used by takeoff
+float32 yaw
+float32 x
+float32 y
+float32 z
```

Comment:
geometry_msgs/Vector3 (vector) or geometry_msgs/Point (point in the 3D space)?
I would go for:
```
float32 min_pitch	# used by takeoff
float32 yaw
geometry_msgs/Point position
```